src/adapter/src/coord.rs

```diff
@@ -2507,7 +2549,8 @@ impl Coordinator {
                             timeline: _,
                         } => source_desc(data_source_desc, &table.desc),
                     };
-                    collections.push((id, collection_desc));
+                    collections
+                        .extend(table.global_ids().map(|gid| (gid, collection_desc.clone())));
```

Comment:
How can we just use the same `collection_desc` for all versions? Couldn't it be different across different versions?

Reply:
Great catch! This was definitely a bug, updated!